### PR TITLE
Adds documentation related error and latency analysis of analytics monitoring dashboard.

### PR DESCRIPTION
**Pull request by Ketharan**
## Purpose
Adds documentation related error and latency analysis of analytics monitoring dashboard.

## Automation tests
 - Integration tests
Tested the rendering locally

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? N/A
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Comment by ruks** [comment]
Can we organize the content, same as the order of pages?

**Comment by Ketharan** [comment]
Fixed with https://github.com/wso2/docs-apim/pull/1707/commits/2bd1781a9483724463111e1c391948cc35f07bde

**Comment by Mariangela** [comment]
@ruthryi , please review this linguistically

File: en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md [hunk 1]
```diff
@@ -33,3 +33,58 @@ Example:
 
 Multiple filtering level support similar to the previous behavior and API, Version and resource level filtering can be applied to drill down to resource level. An issue or pattern can be identified in each version and resource. Similar to the previous chart, the required dimensions can be enabled or disabled to clearly identify the lines. 
 
+## API Error Analysis 
+
+Error page allows users to identify faulty and erroneous API(s) and isolate the cause for their behavior. API Errors or Faulty behaviors include the throttled counts, 4xx, and 5xx responses from the API backend. Widgets on this page are helpful to identify the API(s) and Application(s) with the most errors for each mentioned type. Further, provide the summary of those errors per API and Application, and show how it varied historically.
```
[the remark this answers is not in the document]
Comment (Ketharan) [reply en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md:7]: > what do you mean by error page? like a 404?

Not exactly, we divide a dashboard into several pages and cover specific things in each. Here the Error page is dedicated to visualize the statistics related to API and application errors (4xx, 5xx, throttling).

[the remark this answers is not in the document]
Comment (Ketharan) [reply en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md:7]: > you mean the response from a error includes the throttled count and the HTTP 4xx or 5xx error response?

Kind of other way around. As mentioned earlier this page is dedicated to API and Application errors. When we send a request to gateway that might throttle out the request, might receive 4xx or 5xx responses from API backend and some other faults (Connection issues with backend) also can occur. The term error covers the mentioned type of responses.

[the remark this answers is not in the document]
Comment (Ketharan) [reply en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md:7]: > what are the types? APIs and Applications?
Here the 'types' refer the error types namely 4xx, 5xx, faults, and throttling. These can be viewed per API, per Application, and for a selected API and Application pair.

[the remark this answers is not in the document]
Comment (Ketharan) [reply en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md]: > and isolate the cause for their behavior -> and isolate the cause.

updated with https://github.com/wso2/docs-apim/pull/1707/commits/ce69c0682c6950e99bb1e0316f68d809e7a5abef

[the remark this answers is not in the document]
Comment (Ketharan) [reply en/docs/learn/analytics/viewing-api-statistics/monitoring-dashboard.md]: > Further, provide the summary of those errors per API and Application, and show how it varied historically. -> A historical summary could also be viewed per API or per Application,

Updated with https://github.com/wso2/docs-apim/pull/1707/commits/ce69c0682c6950e99bb1e0316f68d809e7a5abef

**Review by shaniR — APPROVED** [review]

**Pull request merged**